knowledge-base/font-icons-not-rendering.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Font Icons do not render in Telerik UI for Blazor 4.6
+description: Font Icons do not render in Telerik UI for Blazor 4.6
+type: troubleshooting
+page_title: Font Icons do not Render in Telerik UI for Blazor 4.6.0
+slug: icon-kb-font-icons-not-rendering
+position: 
+tags: telerik, blazor, fonticon, font, icon
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>4.6</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>Progress® Telerik® UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+After upgrading to the 4.6.0 version of the Telerik UI for Blazor suite the Font Icons in my application are not longer showing (rendering). 
+
+## Possible Cause
+
+Each Telerik UI for Blazor version has a dependency on a specific version of the Kendo UI Themes. As part of the [7.0.0 version of the Kendo UI Themes](https://github.com/telerik/kendo-themes/blob/develop/CHANGELOG.md#breaking-changes), the Font Icons were detached from the themes distribution.
+
+The 7.0.0 version of the Kendo UI Themes is a dependency in Telerik UI for Blazor 4.6.0; thus, the font icons are missing.
+
+## Solution
+
+To keep using Font icons in your application use one of the two approaches:
+
+* Use the dedicated Telerik UI for Blazor CDN in your application:
+
+````_Host.cshtml
+ <link href="https://blazor.cdn.telerik.com/blazor/4.6.0/kendo-font-icons/font-icons.css" rel="stylesheet" type="text/css" />
+````
+````index.html
+ <link href="https://blazor.cdn.telerik.com/blazor/4.6.0/kendo-font-icons/font-icons.css" rel="stylesheet" type="text/css" />
+````
+
+* Download the [Telerik UI for Blazor package]({%slug installation/msi%}#how-to-download-the-automated-installer) and reference the static assets for the Font Icons:
```

Comment:
The download step should be unnecessary. Customers already have the package.

Reply:
I have added the Trial version of the static assets reference.

I have added this step to "cover all bases". I would like it to stay in the article.